### DISCUSSION TITLE
Expectation for getting fluid in and out of inline stations

### DISCUSSION THREAD

**Discussion by Ray-Arpin** [Opinionated-Blueprints/10-Books-Full-of-Rails#55] [Q&A] [answered]
What is the expectation for getting liquids into and out of these stations? For the solid inline stations, there are output and input underground conveyors that make it obvious how the author expects the materials to get into the stations and onto the trains. There is nothing like that for the liquid stations, especially for the 1-1-0 Trains 3 Stations, where (I would assume) the underground piping location is covered by the automation blocks.
![Screenshot_20241119_174452](https://github.com/user-attachments/assets/608a8de1-5875-448d-94a4-d323c9bdb465)
![Screenshot_20241119_174548](https://github.com/user-attachments/assets/259956c3-391c-4b9f-a81c-09b4fb3de22d)

**Comment by Bocian-1** [comment, marked as answer]
Routing a bunch of belts in the compact space is difficult so it made sense to have L/R versions with them routed out. For fluid it's trivial, so I prefer to avoid bloat. You can route it very easily wherever you like
